launch/launch/actions/include_launch_description.py

```diff
@@ -125,6 +128,21 @@ def get_sub_entities(self):
         ret = self.__launch_description_source.try_get_launch_description_without_context()
         return [ret] if ret is not None else []
 
+    def _try_get_arguments_names_without_context(self):
+        try:
+            context = LaunchContext()
+            return [
+                perform_substitutions(context, normalize_to_list_of_substitutions(arg_name))
+                for arg_name, arg_value in self.__launch_arguments
+            ]
+        except Exception as exc:
+            self.__logger.debug(
+                'Failed to get launch arguments names for launch description: '
+                f'{self.__launch_description_source.location}'
+                f'exception: {str(exc)}'
```

Comment:
I think this will result in the word 'exception' and the path getting concatenated. What about this (two colons was kind of weird):

```suggestion
                'Failed to get launch arguments names for launch description '
                f"'{self.__launch_description_source.location}', "
                f'with exception: {str(exc)}'
```

Or at least something like this:

```suggestion
                'Failed to get launch arguments names for launch description: '
                f'{self.__launch_description_source.location}, '
                f'exception: {str(exc)}'
```

Reply:
Right :man_facepalming:.
I applied the first suggestion.